salt/states/linux_acl.py

```diff
@@ -25,6 +25,9 @@
          - perms: rwx
 '''
 
+# Import python libs
+import os
```

Comment:
Could you move this down below the `absolute_import` statement below it? The `absolute_import` statement needs to be at the top. Thanks.